src/autorest-core/test/plugin.ts

```diff
@@ -57,7 +57,7 @@ import { LoadLiterateSwagger } from "../lib/pipeline/swagger-loader";
       }
     });
     assert.strictEqual(await autoRest.Process().finish, true);
-    const expectedNumErrors = 7;
+    const expectedNumErrors = 2;
```

Comment:
Yup we fixed some bugs in oav like `integers` in query param, `encoding` checks and added support for polymorphic models that's why some of the errors disappeared. 

Ref: https://github.com/Azure/oav/issues?q=is%3Aissue+is%3Aclosed